locksmith/src/utils/gasSettings.ts

```diff
@@ -42,24 +42,29 @@ export const getGasSettings = async (network: number): Promise<GasSettings> => {
   try {
     feedata = await provider.getFeeData()
   } catch (error) {
-    // do nothing
+    console.log(`Could not retrieve fee data from ${network}, ${error}`)
   }
 
   if (feedata) {
-    const { gasPrice, maxFeePerGas, maxPriorityFeePerGas } = feedata
+    const { gasPrice, maxFeePerGas } = feedata
 
     // We double to increase speed of execution
     // We may end up paying *more* but we get mined earlier
+    if (maxFeePerGas) {
```

Comment:
Some networks will trigger errors when both `maxFeePerGas` and `gasPrice` is sent...